pom.xml

```diff
@@ -52,7 +52,7 @@ under the License.
 	<repositories>
 		<repository>
```

Comment:
The repo is actually disabled because both `snapshots` and `releases` are set to `false` since: https://github.com/apache/flink/pull/19804